libs/core/src/lib/select/select.component.html

```diff
@@ -47,7 +47,7 @@
         [attr.aria-haspopup]="!(this.readonly || this.disabled)"
         [attr.aria-required]="required"
         [attr.title]="triggerValue"
-        [attr.aria-label]="ariaLabel"
+        [attr.aria-label]="ariaLabel || ('platformSelect.selectOptionLabel' | fdTranslate)"
```

Comment:
Sorry for the confusion, I was referring to the operator here. Because you are setting ariaLabel in the component to the hardcoded english string, the translation will never be used here